packages/resolvers/default/src/DefaultResolver.js

```diff
@@ -132,7 +132,18 @@ class NodeResolver {
     // Resolve the module directory or local file path
     let module = await this.resolveModule(filename, parent, isURL);
 
-    if (module.moduleDir) {
+    if (module.moduleName && process.versions.pnp != null) {
```

Comment:
I think we should make a separate `PnPResolver` and add it to the pipeline in the default config. If it doesn't resolve anything, we'll fall back to the default resolver. I'm working on breaking up the default resolver into multiple pieces anyway, so better to keep it separate if possible.

Reply:
That's what I wanted to do at first as well, but that would only work with some more resolvers (in that order):
- AliasAndTildeResolver
- PNPResolver
- NodeResolver